xml/art_amd-sev.xml

```diff
@@ -350,6 +350,40 @@
   </para>
  </sect1>
 
+ <sect1 xml:id="sec-amd-sev-vminst">
+  <title>VM installation</title>
+  <para>
+   <command>virt-install</command> supports installation of SEV and SEV-ES
+   virtual machines. In addition to your standard installation parameters,
+   provide <command>virt-install</command> with options to satisfy the VM
+   requirements and the <literal>--launchSecurity</literal> option.
+  </para>
+  <para>
+   The following example starts a network installation of a SLES15 SP4 virtual
+   machine protected with SEV-ES.
+  </para>
+  <screen>
+   virt-install --name sles15sp4-sev-es --location http://192.168.0.1/install/sles15sp4/x86_64 --disk size=20,target.bus.scsi --controller type=scsi,model=virtio-scsi,driver.iommu=on --network=bridge=br0,model=virtio,driver.iommu=on --vcpus 4 --memory 4096 --noautoconsole --events on_reboot=destroy --machine q35 --memtune hard_limit=4563402 --launchSecurity sev,policy=0x07 <co xml:id="sec-amd-sev-ex-launchsec"/> --boot firmware=efi <co xml:id="sec-amd-sev-ex-fw"/> --vnc --serial pty
+  </screen>
+  <calloutlist>
+   <callout arearefs="sec-amd-sev-ex-launchsec">
+    <para>
+     The <literal>launchSecurity</literal> option specifies the type and policy to be
+     enforced by the SEV firmware. The policy setting is described in
+     <xref linkend="table-guestpolicy"/>.
```

Comment:
I would mention again here that bit 2 of the policy enables ES support. This makes it more accessible to the reader, even if the information is already in the table blow.

Reply:
Ok. I've adjusted the corresponding patch.